pom.xml

```diff
@@ -1028,7 +982,6 @@
 				</property>
 			</activation>
 			<modules>
-				<module>neo4j-migrations-test-results</module>
```

Comment:
This is required, otherwise central will fail. see comment above in pom.

Reply:
I think Intellij removed it, I have no recollection of removing it 🤔